### ISSUE TITLE
bin/setup failures

### ISSUE THREAD

**Issue by louise-r-blue** [ackama/rails-template#12]
Following is an intermittent issue. Having recently updated my macOS to Catalina am having a issue with setting up the ackama rails template in using both `rails 6.0.0` & ` ruby 2.6.2` and also with `rails 6.0.1` and `ruby 2.6.5`

The issue seems to be that the `bin/setup` script does not run automatically . In this particular instance the errors point to the gems `Bootsnap` and `ffi` not being found.

after running `gem pristine bootsnap 1.4.5` 
and
`gem pristine ffi 1.11.3`
I ran `bin/setup`  (everything ran as I would expect)
and then I ran 
`brew update-reset`  

After running last command things seem to be working as normal until the `bin/rake db: create:all` is run the next error happens.

`Sprockets::Railtie::ManifestNeededError: Expected to find a manifest file in`
``app/assets/config/manifest.js``

**Comment by louise-r-blue** [comment]
this is a continuation of the above but don't know if it is related.

When trying to make my first commit... (this is with just trying to set up ackama rails template. No new code added) 

`Errors on modified lines:
/Users/louiserobinson-blue/Documents/workplace/yaaw/Dockerfile:16:    apt-get install -y curl 
/Users/louiserobinson-blue/Documents/workplace/yaaw/Dockerfile:46:# standalone provided valid configuration. When running in docker-compose, these 
/Users/louiserobinson-blue/Documents/workplace/yaaw/app/views/layouts/application.html.erb:30:    
/Users/louiserobinson-blue/Documents/workplace/yaaw/app/views/layouts/application.html.erb:33:    
/Users/louiserobinson-blue/Documents/workplace/yaaw/docker-compose.yml:8:    env_file: 
/Users/louiserobinson-blue/Documents/workplace/yaaw/docker-compose.yml:21:  app: 
/Users/louiserobinson-blue/Documents/workplace/yaaw/docker-compose.yml:23:    links: 
/Users/louiserobinson-blue/Documents/workplace/yaaw/docker-compose.yml:42:  pgdata: 
/Users/louiserobinson-blue/Documents/workplace/yaaw/spec/rails_helper.rb:41:  
/Users/louiserobinson-blue/Documents/workplace/yaaw/spec/rails_helper.rb:44:  

Check Gemfile dependencies..............................[BundleCheck] OK
Analyze with RuboCop........................................[RuboCop] FAILED
Errors on modified lines:
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:1:1: C: Style/FrozenStringLiteralComment: Missing magic comment `# frozen_string_literal: true`.
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:1:8: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:4:6: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:6:5: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:6:14: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:7:1: C: Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `puma` should appear before `rails`.
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:7:5: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:7:13: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:8:1: C: Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `pg` should appear before `puma`.
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:8:5: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:8:11: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:9:1: C: Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `dotenv-rails` should appear before `pg`.
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:9:30: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:10:1: C: Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `bootsnap` should appear before `dotenv-rails`.
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:10:5: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:10:17: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:11:5: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:12:1: C: Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `redis` should appear before `sassc-rails`.
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:12:5: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:12:14: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:13:5: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:13:16: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
/Users/louiserobinson-blue/Documents/workplace/yaaw/Gemfile:14:5: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols`

**Comment by louise-r-blue** [comment]
There is many of the root files missing

```
.browserslistrc
rubocop-https---bitbucket-org-rabidtech-rabid-dotfiles-raw-master--rubocop-yml
.rubocop.yml
babel.config.js
postcss.config.js
yarn.lock

```
so it may be a Webpacker thing

**Comment by louise-r-blue** [comment]
Latest: Adding `gem "sprockets", "~> 3.0"` to gem file and running `bundle update sprockets` seemed to fix the `Sprockets::Railtie::ManifestNeededError: Expected to find a manifest file in `app/assets/config/manifest.js` issue.

**Comment by eoinkelly** [comment]
Fixed by PR #13
